### PR TITLE
En_Bomjimb

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c [hunk 1]
```diff
@@ -40,79 +83,868 @@ const ActorInit En_Bomjimb_InitVars = {
...
+        }
+    }
+
+    this->actor.world.rot.y = this->unk_2D6 + this->unk_2D4;
+
+    if (func_800C99AC(&globalCtx->colCtx, sp58, sp5C)) {
+        s16 temp = BINANG_SUB(this->actor.world.rot.y, this->actor.yawTowardsPlayer - 0x8000);
+
+        if (temp < 0) {
+            this->unk_2D4 += -0x1000;
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c:619]: Would this match?
```suggestion
            this->unk_2D4 -= 0x1000;
```

**Review by AngheloAlf — APPROVED** [review]
Nice work.
Just a minor suggestion, but it doesn't prevent me to approve the PR

**Review by jpburnett — APPROVED** [review]
nice work!

File: src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c [hunk 2]
```diff
@@ -40,79 +83,868 @@ const ActorInit En_Bomjimb_InitVars = {
...
+    return false;
+}
+
+s32 func_80C013F0(EnBomjimb* this, GlobalContext* globalCtx) {
+    Actor* sp1C;
+
+    if (this->collider.base.ocFlags1 & OC1_HIT) {
+        sp1C = this->collider.base.oc;
+
+        if ((sp1C->id == ACTOR_OBJ_KIBAKO) && (D_80C03170 == 0) && (sp1C->update != NULL) &&
```
[the remark this answers is not in the document]
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c:262]: Haven't been doing this anywhere else. I think it'd look out of place just changing the parens here.

[the remark this answers is not in the document]
Comment (AngheloAlf) [reply src/overlays/actors/ovl_En_Bomjimb/z_en_bomjimb.c:262]: You should at least change that 0 for a proper NULL tho

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**